src/index.js

```diff
@@ -35,21 +35,15 @@ import {
 
 // Wrappers
 import Web3Wrapper from './wrappers/web3'
-import { withAccount } from './wrappers/web3'
+import Loadable, { RESTORER } from './wrappers/loadable'
```

Comment:
`RESTORER` is the.worrrrst.

But really so is the loadable thing, and it's all my fault.

Reply:
Yeah I'm going to do some reading on my frontend patterns, next chance I get. At least it encapsulates the restoration rather robustly, albeit verbosely.